src/devices/Ina219/Ina219.cs

```diff
@@ -241,7 +241,9 @@ public void SetCalibration(ushort calibrationValue, float currentLsb = 1.0F)
             WriteRegister(Ina219Register.Calibration, calibrationValue);
         }
 
-        /// <inheritdoc/>
+        /// <summary>
```

Comment:
Does styleCop not like `<inheritdoc />` ?

Reply:
The problem is that StyleCop does not like `<inheritdoc/>` when there's nothing to inherit from. This is not overloading anything.

Reply:
you are right! I thought this was the regular Dispose method and not the one that takes in the bool.